tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -202,11 +204,15 @@ Remove publication owner access
     [Tags]  HappyPath
     user waits until element is enabled  css:[name="selectedPublicationId"]
     user scrolls to element  css:[name="selectedPublicationId"]
-    user clicks element  testid:remove-publication-role-${PUBLICATION_NAME}
-    user waits until page does not contain loading spinner
+    # NOTE: The below wait is to prevent a transient failure that occurs on the UI test pipeline due to the DOM not being fully rendered which 
```

Comment:
Can this not be achieved with:

```
user waits until testid is visible       remove-publication-role-${PUBLICATION_NAME}
```

rather than the wait?

Reply:
Afraid not. I tried the above and various other waits but it seems that the only way to stop CI failures is to add  `Sleep`

Reply:
No worries!